lib/balance-tx/lib/internal/Internal/Cardano/Write/Tx/Balance/Size/Selection.hs

```diff
@@ -22,7 +22,7 @@
 --
 -- Module containing logic relating to size estimation as needed, mainly for
 -- the purpose of coin selection.
-module Internal.Cardano.Write.Tx.SizeEstimation
+module Internal.Cardano.Write.Tx.Balance.Size.Selection
```

Comment:
I must admit I do prefer the original term `Tx.SizeEstimation` here, because it really does convey to the reader that this part of the hierarchy relates to estimating the size of a transaction.

But assuming we have to fit within a `Tx.Balance.Size` hierarchy, then I think we should avoid using the term `Selection` here, if possible, as:
- the term `Selection` is rather overloaded.
- this module doesn't deal with objects of type `Selection`.

However:
- this module does define and export the `TxSkeleton` type.
- size and cost estimation functions are defined in terms of `TxSkeleton` objects.

Perhaps we could rename this module to `TxSkeleton` instead?
So:
```suggestion
module Internal.Cardano.Write.Tx.Balance.Size.TxSkeleton
```